apps/gadgetron/Server.cpp

```diff
@@ -1,15 +1,20 @@
 
 #include <boost/asio.hpp>
 
+#include "../clients/gadgetron_ismrmrd_client/gadgetron_ismrmrd_client.cpp"
 
 #include <Context.h>
-
+#include "GadgetronSlotContainer.h"
 #include "log.h"
 
-#include "Server.h"
+#include "Gadget.h"
+#include "../../network/messages/ConfigurationReader.h" // gotta find out how to include this file using submodule
 #include "Connection.h"
+#include "Gadget.h"
+#include "Server.h"
 #include "connection/SocketStreamBuf.h"
 #include "system_info.h"
+#include "LegacyACE.h"
```

Comment:
ACE is deprecated, so it should not be reintroduced.

Reply:
The LegacyACE.h and LegacyACE.cpp implement ACE structures and methods, so Gadgets written referencing ACE old structures can be used with minimal changes. However, all new code should not use these and should directly use the new structures. At some point the Legacy structures will be dropped, they are only included to allow people time to migrate custom gadgets.